tests/test_virtualenv.py

```diff
@@ -24,12 +24,15 @@
 from unittest import mock
 
 import pytest
+import virtualenv
+from packaging import version
 
 import nox.virtualenv
 
 IS_WINDOWS = nox.virtualenv._SYSTEM == "Windows"
 HAS_CONDA = shutil.which("conda") is not None
 RAISE_ERROR = "RAISE_ERROR"
+VIRTUALENV_VERSION = virtualenv.version.version
```

Comment:
That's only available in recent versions of virtualenv, using `hatch-vcs`.

In older versions, the path was: `virtualenv.__version__` (which is still available today)